standalone/package.json

```diff
@@ -75,7 +98,6 @@
 		"@guardian/libs": ">6"
 	},
 	"dependencies": {
-		"@guardian/commercial-core": "^4.25.0",
```

Comment:
Should commercial-core still be a dependency rather than a development dependency?

Reply:
I'd say it's a dev dependancy? As it's only needed for the bundle build not at runtime. That's how i think about it anyway, happy to change though

Reply:
Maybe I'm thinking about it wrong, but I considered any code that gets pulled into the standalone bundle a dependency (so 3rd party stuff like fastdom and our stuff like CMP), since it'll end up being used at runtime? I'm not too sure now, so I'll leave it up to you.